commit 50050b96b369f0d518cbe4481989fd066fc80ef7
Author: Michael Pyne <mpyne@kde.org>
Date:   Mon Aug 19 19:32:12 2013 -0400

    kio_ui: Fix crash for Slaves without a Job.
    
    Backport of 8dbb27a6 from master to 4.11.1.
    
    Reviewed and ack'ed by adawit.

diff --git a/kio/kio/usernotificationhandler.cpp b/kio/kio/usernotificationhandler.cpp
index 8605198..4c05706 100644
--- a/kio/kio/usernotificationhandler.cpp
+++ b/kio/kio/usernotificationhandler.cpp
@@ -76,7 +76,7 @@ void UserNotificationHandler::processRequest()
 
         if (m_cachedResults.contains(key)) {
             result = *(m_cachedResults[key]);
-        } else {
+        } else if (r->slave->job()) {
             SimpleJobPrivate* jobPrivate = SimpleJobPrivate::get(r->slave->job());
             if (jobPrivate) {
                 result = jobPrivate->requestMessageBox(r->type,
